packages/app-store/routing-forms/pages/router/getServerSideProps.ts

```diff
@@ -131,25 +140,24 @@ export const getServerSideProps = async function getServerSideProps(
 
   const decidedAction = matchingRoute.action;
 
-  const { createContext } = await import("@calcom/trpc/server/createContext");
-  const ctx = await createContext(context);
-
-  const { default: trpcRouter } = await import("@calcom/app-store/routing-forms/trpc/_router");
-  const caller = trpcRouter.createCaller(ctx);
```

Comment:
Avoid using another `trpcHandler` and instead use the utility directly.

Also, needed to do this so that I can easily pass on already queried `form` to `handleResponse` and avoid repeat querying of same data